app/src/main/java/com/doctor/yumyum/data/remote/RemoteDataSource.kt

```diff
@@ -0,0 +1,19 @@
+package com.doctor.yumyum.data.remote
+
+import android.telecom.Call
+import com.doctor.yumyum.common.network.RetrofitClient
+import com.doctor.yumyum.data.model.signUpModel
+import com.doctor.yumyum.data.remote.api.AuthCreationService
+import okhttp3.ResponseBody
+import retrofit2.Response
+
+interface RemoteDataSource {
+    fun postAuthCreation(signUpModel: signUpModel): retrofit2.Call<ResponseBody>
```

Comment:
```suggestion
    suspend fun postAuthCreation(signUpModel: signUpModel): ResponseBody
```
network 통신할 때도 코루틴 사용해도 괜찮을 듯 ㅎㅎ

Reply:
코루틴의 매력을 알아버린거 같아요..